core/src/test/resources/org/bitcoinj/script/tx_invalid.json

```diff
@@ -109,5 +109,9 @@
 [[["ad503f72c18df5801ee64d76090afe4c607fb2b822e9b7b63c5826c50e22fc3b", 0, "0x21 0x027c3a97665bf283a102a587a62a30a0c102d4d3b141015e2cae6f64e2543113e5 CHECKSIG NOT"]],
 "01000000013bfc220ec526583cb6b7e922b8b27f604cfe0a09764de61e80f58dc1723f50ad0000000000ffffffff0101000000000000002321027c3a97665bf283a102a587a62a30a0c102d4d3b141015e2cae6f64e2543113e5ac00000000", "P2SH"],
 
+["A transaction with a non-standard DER signature."],
```

Comment:
I take it you created these test cases yourself? That's great!

There is only one issue with that: we sync these .json files from Bitcoin Core on a regular basis. So maybe you can submit your testcases to Core too?